### PR TITLE
NMS-10538: correct capitalization of hrSWRun* in docs

### PR DESCRIPTION
**Pull request by mmahacek**
### All Contributors

* [X] Have you read and followed our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [X] Have you [made an issue in the OpenNMS issue tracker](https://issues.opennms.org)?<br>If so, you should:
  1. update the title of this PR to be of the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
* [X] Have you made a comment in that issue which points back to this PR?
* [X] Have you updated the JIRA link at the bottom of this comment to link to your issue?
* [X] If this PR targets a `foundation-*` branch, does it avoid changing files in `$OPENNMS_HOME/etc/`?

### Pull Request Process

One or more reviewers should be assigned to each PR.

If you know that a particular person is subject matter expert in the area your PR affects, feel free to assign one or more reviewers when you create this PR, otherwise reviewers will be assigned for you.

If you have made additions or changes to the documentation, or if you _need_ documentation for these code changes, please make sure a technical writer has looked it over.

Once the reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the Order of the Green Polo) you are welcome to merge the PR.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-10538

### CODE REVIEW

**Review by Bonrob2 — APPROVED** [review]
LGTM.

File: docs/modules/operation/pages/service-assurance/monitors/HostResourceSwRunMonitor.adoc [hunk 1]
```diff
@@ -2,8 +2,8 @@
 = HostResourceSwRunMonitor
 
 This monitor test the running state of one or more processes.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/service-assurance/monitors/HostResourceSwRunMonitor.adoc:4]:
```suggestion
This monitor tests the running state of one or more processes.
```

File: docs/modules/operation/pages/service-assurance/monitors/HostResourceSwRunMonitor.adoc [hunk 2]
```diff
@@ -2,8 +2,8 @@
...
-It does this via SNMP by inspecting the _hrSwRunTable_ of the http://www.ietf.org/rfc/rfc2790[HOST-RESOURCES-MIB].
-The test is done by matching a given process as _hrSwRunName_ against the numeric value of the _hrSwRunState_.
+It does this via SNMP by inspecting the _hrSWRunTable_ of the http://www.ietf.org/rfc/rfc2790[HOST-RESOURCES-MIB].
+The test is done by matching a given process as _hrSWRunName_ against the numeric value of the _hrSWRunState_.
```
Comment (Bonrob2) [inline docs/modules/operation/pages/service-assurance/monitors/HostResourceSwRunMonitor.adoc:6]:
```suggestion
The test matches a given process as _hrSWRunName_ against the numeric value of the _hrSWRunState_.
```

**Review by Bonrob2 — APPROVED** [review]
LGTM, except for some italics which I was unable to offer update suggestions. I believe you addressed them in the next commit.

**Review by Bonrob2 — APPROVED** [review]
LGTM!

**Review by indigo423 — APPROVED** [review]
Thank you for the improvemnts ship it :) 🚢

**Pull request merged**